### DISCUSSION TITLE
When is v1 coming?

### DISCUSSION THREAD

**Discussion by tontonsb** [dedoc/scramble#595] [General]
Are there any plans to tag the version 1.0.0 any time soon?

**Comment by romalytvynenko** [comment]
@tontonsb yes!

I was going to make a public roadmap. Just hesitant about breaking changes for v1. 

I would really want to change `OpenApi` classes' structure (make them simple, plain, and make sure that they fully cover OpenAPI spec), but maybe this is to much pain for everyone using the package.

I'd also want to use interfaces for extensions. Also implementing Scramble PRO showed me a missing pieces of type to schema extensions. 

So yeah. Wondering what are your thoughts on this? And why v1 is important personally for you? 😇

**Reply by tontonsb** [reply]
For me it's not so much about "v1" per se, more about the note in the description:

> Warning Package is in early stage. It means there may be bugs and API is very likely to change.

I'm looking for a little more stability and predictability before introducing this in some long-term projects where we don't want a burden of constantly following package upgrades and API changes.

**Reply by romalytvynenko** [reply]
@tontonsb gotcha. Are you using your own extensions by the way?
